app/components/application/AppEndpointCard/AppEndpointCard.tsx

```diff
@@ -159,6 +165,85 @@ export default function AppEndpointCard({ app, blockchains }: AppEndpointProps)
               />
             )
           })}
+        <Grid
+          align="start"
+          justify="space-between"
+          sx={{
+            margin: "2em 0 0",
+            paddingTop: "2em",
+            borderTop: "1px solid #2F373E",
+          }}
+        >
+          <Box
+            sx={{
+              width: "100%",
+            }}
+          >
+            <Grid
+              align="center"
+              justify="space-between"
+              sx={{
+                margin: "0 0 2em 0",
+              }}
+            >
+              <Grid align="center" m={0}>
+                <Text m="0 0.5em 0 0" size="lg" weight={600}>
+                  Your Data API
+                </Text>
+                <Text m="0 0.5em 0 0" size="sm" weight={400}>
+                  (powered by Covalent)
+                </Text>
+                <HelpTooltip
+                  label={
+                    "The following API key can be used to authenticate against the Covalent API. When authenticating, use the API key as the username and a blank password."
+                  }
+                />
+              </Grid>
+              <Button className="pokt-button-outline" color="blue" variant="outline">
```

Comment:
The way this button wraps the anchor tag makes it so not every click takes you to the url. Please update this something like the code below instead.

<Anchor component={Button} href="" />